.github/workflows/lambda_zip.yaml

```diff
@@ -0,0 +1,58 @@
+name: release
+
+on:
+  release:
+    types:
+      - created
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    # create the zip file
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v2
+
+      - name: Build Lambda Function
+        run: |
+          zip lambda.zip rolling-restart.py
+          ls
```

Comment:
no need `ls`

Reply:
You noticed this on my PR but not on Roberto's? lol

Reply:
```suggestion
```